core/src/main/java/com/datastax/oss/driver/internal/core/metrics/DropwizardMetricsFactory.java

```diff
@@ -49,9 +50,9 @@
   @Nullable private final Metrics metrics;
   private final SessionMetricUpdater sessionUpdater;
 
-  public DropwizardMetricsFactory(InternalDriverContext context) {
+  public DropwizardMetricsFactory(DriverContext context) {
     this.logPrefix = context.getSessionName();
-    this.context = context;
+    this.context = (InternalDriverContext) context;
```

Comment:
This change can be reverted as we're not exposing a config option to instantiate a MetricsFactory